libnavigation-base/src/main/java/com/mapbox/navigation/base/route/Router.kt

```diff
@@ -1,9 +1,25 @@
 package com.mapbox.navigation.base.route
 
 import com.mapbox.geojson.Point
+import com.mapbox.navigation.base.route.model.Route
 
 interface Router {
 
+    @Deprecated("use [getRoute(Point, List<Point>, RouteListener)] instead")
```

Comment:
Why do we have it in new modules? I thought if it is new module, there shouldn't be any deprecated methods. WDYT?

Reply:
it's only for us, till v1.0. have a look at comment 😉